unit-tests/src/test/scala/java/util/function/BiConsumerTest.scala

```diff
@@ -25,14 +30,12 @@ class BiConsumerTest {
     assertTrue(result == 4)
   }
 
-  @Ignore("andThen doesn't work currently")
   @Test def biConsumerAndThen(): Unit = {
-    // result = 0
-    // assertTrue(result == 0)
-    // addT.andThen(timesT).accept(2, 3)
+    assertTrue(result == 0)
```

Comment:
Given the `@Before`, (clever that!), this seems excessive & redundant.  Yes, some day somebody could remove the `@Before` but then they will get hosed/allerted  by the 
test failing.